README.md

```diff
@@ -1,8 +1,62 @@
 # ember-cli-preprocessor-registry
 
-Used internally by Ember CLI to provide a registry of preprocessors. The main types
+Used by Ember CLI to provide a registry of preprocessors. The main types
 used throughout the system are `css`, `template`, `js`.
 
+## Addon Usage
+
+You can access both your own addon's and your parent's (whichever item is including you)
+registry via the `setupPreprocessorRegistry` hook in your addon's `index.js`.
+
+Example:
+
+```js
+module.exports = {
+  name: 'special-js-sauce',
+  
+  setupPreprocessorRegistry(type, registry) {
+    registry.add('js', {
```

Comment:
might make sense to add `if (type === 'parent')` or something like that to the example